aws/signer/v4/v4.go

```diff
@@ -92,6 +94,12 @@ type Signer struct {
 	// http://docs.aws.amazon.com/general/latest/gr/sigv4-create-canonical-request.html
 	DisableURIPathEscaping bool
 
+	// The logger to send log message to
+	Logger logging.Logger
+
+	// Enable logging of signed requests
+	LogSigning bool
```

Comment:
Would be helpful to clarify the docs on what will be logged, e.g. string to sign, and canonical signature, etc